kernel/src/main/scala/org/apache/toree/boot/CommandLineOptions.scala

```diff
@@ -193,6 +199,19 @@ class CommandLineOptions(args: Seq[String]) {
     Some(userDefined.asJava)
   }
 
+  private def onlyPositiveOtherwiseDefault(spec: OptionSpec[Int]): Option[Int] = {
```

Comment:
I don't think it is a good idea to silently ignore user input. If the value passed is -1, for example, then the user gets 100ms. If what the user requests isn't possible, then this should fail and print the help message.

Reply:
Sounds good.